src/components/policy/src/policy/src/policy_manager_impl.cc

```diff
@@ -1157,11 +1158,86 @@ void PolicyManagerImpl::OnChangedRemoteControl(
     }
   }
 
+  SendAppPermissionsChanged(device_id, application_id);
+}
+
+void PolicyManagerImpl::SendAppPermissionsChanged(const std::string& device_id,
+      const std::string& application_id) {
   Permissions notification_data;
   GetPermissions(device_id, application_id, &notification_data);
   listener()->OnPermissionsUpdated(application_id, notification_data);
 }
 
+void PolicyManagerImpl::CheckPTUUpdatesChange(
+    const utils::SharedPtr<policy_table::Table> pt_update,
+    const utils::SharedPtr<policy_table::Table> snapshot) {
+  CheckPTURemoteCtrlChange(pt_update, snapshot);
+  CheckPTUZonesChange(pt_update, snapshot);
+  CheckRemoteGroupsChange(pt_update, snapshot);
+}
+
+bool PolicyManagerImpl::CheckPTURemoteCtrlChange(
+    const utils::SharedPtr<policy_table::Table> pt_update,
+    const utils::SharedPtr<policy_table::Table> snapshot) {
+  LOG4CXX_AUTO_TRACE(logger_);
+
+  rpc::Optional<rpc::Boolean>& new_consent =
+    pt_update->policy_table.module_config.country_consent_passengersRC;
+  rpc::Optional<rpc::Boolean>& old_consent =
+    snapshot->policy_table.module_config.country_consent_passengersRC;
+
+  if (!new_consent.is_initialized() && !old_consent.is_initialized()) {
+    return false;
+  }
+
+  bool result = false;
+  if (new_consent.is_initialized() && old_consent.is_initialized()) {
+    result = (*new_consent != *old_consent);
+  } else {
+    bool not_changed_consent1 = !new_consent.is_initialized() && *old_consent;
+    bool not_changed_consent2 = !old_consent.is_initialized() && *new_consent;
+
+    result = !(not_changed_consent1 || not_changed_consent2);
+  }
+
+  if (result) {
+    listener()->OnRemoteAllowedChanged(result);
+  }
+
+  return result;
+}
+
+void PolicyManagerImpl::CheckPTUZonesChange(
+    const utils::SharedPtr<policy_table::Table> pt_update,
+    const utils::SharedPtr<policy_table::Table> snapshot) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  rpc::Optional<Equipment>& old_equipment =
+    snapshot->policy_table.module_config.equipment;
+  rpc::Optional<Equipment>& new_equipment =
+    pt_update->policy_table.module_config.equipment;
+
+  if (!old_equipment.is_initialized() || !new_equipment.is_initialized()) {
+    LOG4CXX_DEBUG(logger_, "No need to reset access, equipment is not inited");
+    return;
+  }
+  // TODO(KKolodiy): change to erasing only affected by PTU changes permissions
```

Comment:
From Google Style:
> The main purpose is to have a consistent TODO that can be searched to find out how to get more details upon request. A TODO is not a commitment that the person referenced will fix the problem. Thus when you create a TODO, it is almost always your name that is given.

I mean you know this context better than me. So your name should be here.